lang/cs/Org.Apache.REEF.Tests/Functional/Bridge/TestFailedEvaluatorEventHandler.cs

```diff
@@ -143,15 +130,13 @@ public void Dispose()
 
             public byte[] Call(byte[] memento)
             {
-                _countdownEvent.Wait();
+                Logger.Log(Level.Info, "Entered FailEvaluatorTask");
+
+                // need to sleep to allow Java code to get heartbeat with information about task
+                Thread.Sleep(100);
```

Comment:
Hmm, so what are the symptoms? The Driver doesn't see that the Task exists before the Evaluator fails?